orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -2209,11 +2209,11 @@ Return true if all elements are true and false otherwise.
         }
       });
 
-  ONNX_CONTRIB_OPERATOR_SCHEMA(YieldOp)
+  ONNX_CONTRIB_OPERATOR_SCHEMA(BreakOp)
```

Comment:
Yield refers to an action that is continuous, like you are driving and then you slowdown to yield way for someone and then continue driving. This was true when threading was in place but with partial graph execution this no longer the case. Partial graph execution is a standalone operation with clear start and end points after which the execution comes to a complete halt, similar to a full graph run. I'm not convinced yield is the most appropiate name and will prefer we chat offline to come to a conclusion. I have already had a conversion on the naming with few folks from the team after which I chose "break". 

---
In reply to: [600911048](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r600911048) [](ancestors = 600911048)

Reply:
I would prefer we not hold up this PR on a debate about naming. Please revert it to Yield, and we can discuss it as a potential refactoring later.

Reply:
@mrry I'm also of the same opinion that we not hold this PR over this debate because every day I pay the tax of fixing merge conflicts :) That being said I will prefer we chat offline on Monday to mutually come to an agreement. May be there is something in your rationale that I may have missed and I will prefer I understand and learn from. It will also be good to have others in the discussion to make sure we are doing the right thing and are being _inclusive_. Thank you, Derek for taking the time to review my change, it's been very beneficial.